src/java.naming/share/classes/com/sun/jndi/ldap/VersionHelper.java

```diff
@@ -59,7 +59,7 @@ public final class VersionHelper {
         // System property to control whether classes is allowed to be loaded from
         // 'javaSerializedData' attribute
         String trustSerialDataSp = getPrivilegedProperty(
-                "com.sun.jndi.ldap.object.trustSerialData", "true");
+                "com.sun.jndi.ldap.object.trustSerialData", "false");
         trustSerialData = "true".equalsIgnoreCase(trustSerialDataSp);
```

Comment:
More of a question - is there a convention in our documentation where we state that the value is case insensitive? Right now, the `module-info.java` uses the literals `false` and `true` and doesn't talk about case insensitivity. Should it?

Reply:
> is there a convention in our documentation where we state that the value is case insensitive?

I think there isn't. Therefore, it will be cleaner to update the property description in `module-info.java` to something like: `"the system property value can be set to {@code true} (case insensitive)"`

And to update the test to check the `case insensitive` clause:

```
* @run main/othervm -Dcom.sun.jndi.ldap.object.trustSerialData=TrUe
*                   RemoteLocationAttributeTest
```

Thanks